daemon/daemon.go

```diff
@@ -27,6 +28,9 @@ func New(cfg *config.EnvConfig) (srv *Daemon, err error) {
 
 	r := mux.NewRouter().StrictSlash(true)
 
+	e := engine.New(cfg)
```

Comment:
Will need to pass this to the HTTP server, so that all the `daemon handlers` have access to the `engine` and can communicate useful information via their HTTP interface.